labs/[1736]_configure-a-firewall-and-a-startup-script-with-deployment-manager/qwiklabs.yaml

```diff
@@ -1 +1 @@
-# delete firewall rule (keep line 15 - 22)
\ No newline at end of file
+# delete firewall rule (keep line 24 - 34)
```

Comment:
line 24-34 its the firewall rule, they need to be deleted (?)

Reply:
yes

Reply:
line 15 - 22 saya hapus 
![okas2](https://user-images.githubusercontent.com/42824788/95772532-17ba5c00-0ce7-11eb-94ae-baa947c16701.png)
![okas1](https://user-images.githubusercontent.com/42824788/95772537-1a1cb600-0ce7-11eb-907b-7865fd6f9693.png)
![okas](https://user-images.githubusercontent.com/42824788/95772542-1ab54c80-0ce7-11eb-9018-acdde29a4e6f.png)


ERROR: (gcloud.deployment-manager.deployments.create) ResponseError: code=412, message=Configuration has invalid format: Unable to parse YAML.

Reply:
Saya hapus line 24 - 34 yang merupakan firewall rule tidak terjadi error dan berhasil  
![okas5](https://user-images.githubusercontent.com/42824788/95772903-ad55eb80-0ce7-11eb-9ef8-5b7617796a57.png)
![okas0](https://user-images.githubusercontent.com/42824788/95772906-ae871880-0ce7-11eb-8b13-f9b0d72102fa.png)
![ikas1](https://user-images.githubusercontent.com/42824788/95772909-af1faf00-0ce7-11eb-943e-5acac9bc47af.png)

Reply:
sebentar,,. yang dihapus firewall rule kan?
line 15-24 tetap di keep

Reply:
mungkin isi `qwiklabs.yaml` bisa diubah jadi

```
imports:
- path: qwiklabs,jinja

resources:
- name: qwiklabs
  type: qwiklabs.jinja
  properties:
    zone: us-central1-a
```